docs/imviz/plugins.rst

```diff
@@ -161,8 +161,8 @@ This plugin only considers pixel locations, not sky coordinates.
 
 .. _aper-phot-simple:
 
-Simple Aperture Photometry
```

Comment:
Haha... I see that you removed "Simple"...

Reply:
renaming is up for debate... but I kinda figured we're not so simple any more and shorter names will be easier to remember/access for the API.

Reply:
I think everyone agreed that it is not simple anymore.